app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    words = command.split()
+    word1 = words[1]
```

Comment:
Consider using more descriptive variable names, such as `source_file` and `destination_file`, to improve code readability.